package.json

```diff
@@ -105,7 +105,7 @@
     "yup": "^0.24.1"
   },
   "dependencies": {
-    "@gooddata/gooddata-js": "^11.14.2",
+    "@gooddata/gooddata-js": "11.16.0",
```

Comment:
We should use caret here. Please use

```
yarn upgrade @gooddata/gooddata-js@^11.16.0
```

to add it. This will probably reduplicate `lodash` so please keep that in mind.

Reply:
aye right i forgot about the caret; will add; although - it still adds the lodash 4.17.15 (that I removed manually from the lockfile before).. so idk - keep the caret & manually remove the extra lodash from lockfile?